styles.css

```diff
@@ -0,0 +1,583 @@
+.overlay {
+  height: 100%;
+  width: 0;
+  position: absolute;
+  z-index: 99;
+  top: 0;
+  left: 0;
+  background-color: rgb(245, 243, 243);
+  overflow-x: hidden;
+  transition: 0.5s;
+}
+
+.overlay-content {
+  position: relative;
+  top: 25%;
+  width: 100%;
+  text-align: center;
+  margin-top: 30px;
+}
+
+.overlay a {
+  padding: 8px;
+  text-decoration: none;
+  font-size: 36px;
+  color: #000;
+  display: block;
+  transition: 0.3s;
+}
+
+.overlay a:hover .overlay a:focus {
+  color: #f1f1f1;
+}
+
+.overlay .closebtn {
+  position: absolute;
+  top: 20px;
+  right: 45px;
+  font-size: 60px;
+}
+
+.overlay-content:hover {
+  color: #ec5242;
+  transition: 0.3s;
+}
+
+.greetings {
+  padding-top: 2.5rem;
+  padding-bottom: 1px;
+  color: #ec5242;
+  font-size: 20px;
+  font-family: inherit;
+}
+
+.title {
+  display: inline-block;
+  color: #ec5242;
+  font-size: 33px;
+  font-weight: 900;
+  font-family: inherit;
+  padding-bottom: 0.5rem;
+}
+
+.title:hover {
+  color: transparent;
+  background-image: url("./images/summit-2.jpeg");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  -webkit-background-clip: text;
+}
+
+.mobile {
+  padding-left: 2rem;
+}
+
+.introduction {
+  padding-top: 1rem;
+}
+
+.brief-intro {
+  border: 3px solid #fff;
+  padding: 0.8rem;
+  min-width: 40%;
+}
+
+.items:hover {
+  outline: 2px solid #d3d3d3;
+  transition: outline 0.15s ease-in-out;
+}
+
+.words-presentation {
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+}
+
+.about:hover {
+  font-family: inherit;
+  color: #ec5242;
+  transition: outline 1s ease-in-out;
+}
+
+.ul-list {
+  font-size: 16px;
+  font-family: inherit;
+}
+
+.date {
+  color: #272a31;
+  padding: 1rem 0 1rem 0;
+  font-family: inherit;
+  font-size: 23px;
+  font-weight: 900;
+}
+
+.national {
+  padding-bottom: 1rem;
+}
+
+main {
+  background-image: url(./images/pattern_bg.png);
+  background-color: #000;
+  background-repeat: repeat;
+  background-size: contain;
+  padding-bottom: 0.1rem;
+  padding-top: 1rem;
+}
+
+.main-program {
+  text-align: center;
+  padding-top: 1rem;
+  font-family: inherit;
+  font-size: large;
+  font-weight: 500;
+  color: #fff;
+  padding-bottom: 0.5rem;
+}
+
+.border {
+  display: flex;
+  border-bottom: 1px solid #ec5242;
+  width: 40px;
+  margin: 0 auto;
+  align-self: center;
+  margin-bottom: 1rem;
+}
+
+.span-1 {
+  color: #ec5242;
+  font-size: 18px;
+}
+
+.d-flex {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 1rem;
+  background: rgba(251, 251, 251, 0.4);
+  color: #fff;
+  padding: 1rem;
+  margin-bottom: 0.5rem;
+}
+
+.base-text {
+  text-align: center;
+  margin: 3rem 2rem;
+  background-color: #ec5242;
+  padding: 2rem;
+  color: #fff;
+  font-size: 18px;
+}
+
+.d-flex-2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 1rem;
+  padding: 1rem;
+  margin-bottom: 0.5rem;
+}
```

Comment:
- [ ] For the Speakers section, kindly align the items in each card to `flex-start` instead of `center` as you have it. This would improve the UI and make your design more identical to that of the original design.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/218115709-eb9bdea3-2bdc-47da-aa30-ba1e193f137e.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/218115919-b7c48628-6bb8-46c2-93e7-8831a21dd05f.png)

#### Hint: `align-items: flex-start;`

Reply:
Done